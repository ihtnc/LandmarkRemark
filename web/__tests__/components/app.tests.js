import React from 'react';
import { shallow } from 'enzyme';
import App from "@components/app"

describe("App component", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot();
  });
});