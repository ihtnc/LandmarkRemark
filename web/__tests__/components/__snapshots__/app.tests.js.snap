// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match latest snapshot 1`] = `
<styled.section>
  <Wrapper />
</styled.section>
`;
